articles/iot-edge/production-checklist.md

```diff
@@ -159,12 +159,12 @@ The IoT Edge agent and IoT Edge hub images are tagged with the IoT Edge version
 
 You know about storing your container images for custom code modules in your private Azure registry, but you can also use it to store public container images such as for the edgeAgent and edgHub runtime modules. Doing so may be required if you have very tight firewall restrictions as these runtime containers are stored in the Microsoft Container Registry (MCR).
 
-Obtain the images with the Docker pull command to place in your private registry. Be aware that you will need to update the images with each new release of IoT Edge runtime.
+Obtain the images with the Docker pull command to place in your private registry. You will need to specify the container version during the pull operation, find the latest container version at container description page as below, and replace the version in the pull command if needed. Be aware that you will need to update the images with each new release of IoT Edge runtime.
 
 | IoT Edge runtime container | Docker pull command |
 | --- | --- |
-| [Azure IoT Edge Agent](https://hub.docker.com/_/microsoft-azureiotedge-agent) | `docker pull mcr.microsoft.com/azureiotedge-agent` |
-| [Azure IoT Edge Hub](https://hub.docker.com/_/microsoft-azureiotedge-hub) | `docker pull mcr.microsoft.com/azureiotedge-hub` |
+| [Azure IoT Edge Agent](https://hub.docker.com/_/microsoft-azureiotedge-agent) | `docker pull mcr.microsoft.com/azureiotedge-agent:1.0` |
+| [Azure IoT Edge Hub](https://hub.docker.com/_/microsoft-azureiotedge-hub) | `docker pull mcr.microsoft.com/azureiotedge-hub:1.0` |
```

Comment:
Since you added the line about checking the list of supported tags and updating the pull operation, what do you think about putting a placeholder in the command instead of a specific tag? Also because we have two supported versions now, `1.1` which is the long-term support branch, and `1.2` which is the latest stable version.

If you think that adds confusion, then the example tag we provide should be at least 1.1, since 1.0 is no longer a supported version.

```suggestion
| [Azure IoT Edge Agent](https://hub.docker.com/_/microsoft-azureiotedge-agent) | `docker pull mcr.microsoft.com/azureiotedge-agent:<VERSION_TAG>` |
| [Azure IoT Edge Hub](https://hub.docker.com/_/microsoft-azureiotedge-hub) | `docker pull mcr.microsoft.com/azureiotedge-hub:<VERSION_TAG>` |
```

Reply:
Yes, totally agree. And the example tag could be the latest one to avoid confusion. I am new to IoT edge module, and just trying the version 1.2 and it works fine for me.

Reply:
Great. If you like the phrasing I suggested, you can accept my suggested changes to merge them into your branch, and then I can approve this pull request.